EarlGrey.xcodeproj/project.pbxproj

```diff
@@ -6,6 +6,20 @@
 	objectVersion = 46;
 	objects = {
 
+/* Begin PBXAggregateTarget section */
+		C88F9A2C1F39A5E000704BFF /* Release */ = {
+			isa = PBXAggregateTarget;
+			buildConfigurationList = C88F9A2F1F39A5E000704BFF /* Build configuration list for PBXAggregateTarget "Release" */;
+			buildPhases = (
+				C88F9A321F39A5F100704BFF /* ShellScript */,
+			);
+			dependencies = (
+			);
+			name = Release;
```

Comment:
This builds a universal framework, although the primary use is for cocaopods, `Release` seems to make more sense because we are building the release build.